build.gradle

```diff
@@ -7,6 +10,7 @@ apply plugin: "maven"
 sourceCompatibility = 1.8
 
 repositories {
+    mavenLocal()
```

Comment:
remove

Reply:
i forget about this, i was using this to deploy locally